atm-fraud/defaults.yaml

```diff
@@ -1,4 +1,4 @@
-kubernetes-app:
+helm-app:
```

Comment:
unrelated?

Reply:
I prefer using the `HelmApp` instead of the abstract `KubernetesApp` class

Reply:
Why though? namespace is a Kubernetes related concept